src/runtime/graph.cc

```diff
@@ -1911,7 +1911,13 @@ void FFModel::deserialize_graph_optimal_view(
       case OP_CONCAT: {
         int legion_axis;
         dez.deserialize(legion_axis);
-        node = get_or_create_concat_node(num_inputs, inputs, legion_axis);
+        std::vector<ParallelTensor> _inputs;
```

Comment:
I don't think this copy is necessary

Reply:
See above comment on `int[]`. I've made the same modification here.

Reply:
I'd recommend the same change as with the other case